protocol/v2/ssv/runner/committee.go

```diff
@@ -250,15 +251,18 @@ func (cr *CommitteeRunner) ProcessConsensus(logger *zap.Logger, msg *spectypes.S
 				zap.String("signing_root", hex.EncodeToString(partialMsg.SigningRoot[:])),
 				zap.String("signature", hex.EncodeToString(partialMsg.PartialSignature[:])),
 			)
-		case spectypes.BNRoleSyncCommittee:
 			validDuties++
+		case spectypes.BNRoleSyncCommittee:
 			blockRoot := beaconVote.BlockRoot
 			partialMsg, err := cr.BaseRunner.signBeaconObject(cr, duty, spectypes.SSZBytes(blockRoot[:]), duty.DutySlot(),
 				spectypes.DomainSyncCommittee)
 			if err != nil {
-				return errors.Wrap(err, "failed signing sync committee message")
+				logger.Warn("failed signing sync committee message for duty, skipping",
+					fields.Validator(duty.PubKey[:]), fields.BeaconRole(duty.Type), zap.Error(err))
+				continue // Skip this duty but continue with others
```

Comment:
this is not needed because SC isn't slashable

Reply:
I'm aware that SC duties aren't slashable. However, the intention here is to ensure that a single error, regardless of whether it's slashable or not, does not impact the entire committee's duties for the slot. This approach aligns with the goal of isolating errors to specific validators without punishing the entire committee.
lmk if you think we should approach this differently

Reply:
if that looks good for @GalRogozinski i have no issues

Reply:
I agree with Oleg here, all slashable or not is actually abstract in this code, so we just want to ensure a single failed signing action doesn't stop all the runner.